devito/core/gpu.py

```diff
@@ -93,7 +93,7 @@ def _normalize_kwargs(cls, **kwargs):
         o['par-nested'] = np.inf  # Never use nested parallelism
         o['par-disabled'] = oo.pop('par-disabled', True)  # No host parallelism by default
         o['gpu-direct'] = oo.pop('gpu-direct', True)
-        o['gpu-fit'] = as_tuple(oo.pop('gpu-fit', None))
+        o['gpu-fit'] = as_tuple(oo.pop('gpu-fit', 'all-fallback'))
```

Comment:
Could you rather read the value from a class attribute? something like `cls.GPU_FIT)